pkg/aws/common_mutator.go

```diff
@@ -26,49 +27,65 @@ type Mutator struct {
 	Logger    micrologger.Logger
 }
 
-func MutateReleaseVersionLabel(m *Mutator, meta metav1.Object) ([]mutator.PatchOperation, error) {
+func MutateLabelFromCluster(m *Mutator, meta metav1.Object, cluster capiv1alpha2.Cluster, label string) ([]mutator.PatchOperation, error) {
```

Comment:
This function becomes generic so we can use it default also the operator version or other labels.